content/src/view.js

```diff
@@ -1324,7 +1324,44 @@ $(window).on('resize.listing', function() {
   }
 });
 
-function updatePaneLinks(pane) {
+
+var searchCache={};
+var prevSearch;
+function updateSearchResults(search,pane){
+    search=search.toLowerCase();
+    if(search==prevSearch)
+        return;
```

Comment:
I prefer to use curly braces always after if, to avoid a common pitfall when adding extra lines.  You may put it all on the same line as the if, if you want to save space.